ddlitlab2024/dataset/models.py

```diff
@@ -87,6 +88,12 @@ class Image(Base):
 
     __table_args__ = (CheckConstraint("stamp >= 0"),)
 
+    def __init__(self, stamp: float, recording_id: int, image: np.ndarray):
+        assert image.dtype == np.uint8, "Image must be of type np.uint8"
+        assert image.shape[2] == 3, "Image must have 3 channels"
+        assert image.ndim == 3, "Image must have 3 dimensions"
```

Comment:
I believe these two lines should be swapped, else shape[2] could be out of bounds